__tests__/index.spec.ts

```diff
@@ -301,43 +318,47 @@ describe('Koa Oas3', () => {
   });
 })
 
-describe('Koa Oas3 with ChowOptions', () => {
-  const mw = oas({
-    file: path.resolve('./__tests__/fixtures/pet-store.json'),
-    endpoint: '/openapi',
-    uiEndpoint: '/openapi.html',
-    validatePaths: ['/pets'],
-    validationOptions: { requestBodyAjvOptions: { allErrors: true } } as ChowOptions
-  });
+// describe.skip('Koa Oas3 with ChowOptions', () => {
+//   let mw: koa.Middleware;
 
-  test('It should coerce values if validation passed', async () => {
-    const ctx = createContext({
-      url: '/pets?limit=10',
-      headers: {
-        'accept': 'application/json'
-      },
-      method: 'GET'
-    });
-    const next = jest.fn();
-    await mw(ctx, next);
-    expect(ctx.oas!.request.query.limit).toBe(10);
-  });
+//   beforeAll(async () => {
+//     mw = await oas({
+//       file: path.resolve('./__tests__/fixtures/pet-store.json'),
+//       endpoint: '/openapi',
+//       uiEndpoint: '/openapi.html',
+//       validatePaths: ['/pets'],
+//       validationOptions: { requestBodyAjvOptions: { allErrors: true } } as ChowOptions
+//     });
+//   })
 
-  test('It should throw ValidationError if validation failed', () => {
-    const ctx = createContext({
-      url: '/pets',
-      headers: {
-        'accept': 'application/json',
-        'content-type': 'application/json'
-      },
-      method: 'POST',
-      body: {
-        id: 1,
-        tag: 'tag'
-      }
-    });
-    const next = jest.fn();
-    return expect(mw(ctx, next)).rejects.toThrow();
-  })
+//   test('It should coerce values if validation passed', async () => {
+//     const ctx = createContext({
+//       url: '/pets?limit=10',
+//       headers: {
+//         'accept': 'application/json'
+//       },
+//       method: 'GET'
+//     });
+//     const next = jest.fn();
+//     await mw(ctx, next);
+//     expect(ctx.oas!.request.query.limit).toBe(10);
+//   });
 
-})
+//   test('It should throw ValidationError if validation failed', () => {
+//     const ctx = createContext({
+//       url: '/pets',
+//       headers: {
+//         'accept': 'application/json',
+//         'content-type': 'application/json'
+//       },
+//       method: 'POST',
+//       body: {
+//         id: 1,
+//         tag: 'tag'
+//       }
+//     });
+//     const next = jest.fn();
+//     return expect(mw(ctx, next)).rejects.toThrow();
+//   })
+
+// })
```

Comment:
Why the comments?

Reply:
Was debugging something. I will put this back.